falcon/request_helpers.py

```diff
@@ -38,6 +42,29 @@ def fget(self):
     return property(fget)
 
 
+def parse_etags(etag_str):
+    """
+    Parse a string of ETags given in an If-Match or If-None-Match header as
+    defined by RFC 7232.
+
+    Args:
+        etag_str (str): A string of ASCII characters placed between double quotes
```

Comment:
Suggested rewording:

> A string of ASCII characters placed between double quotes and may be prefixed by ``W/`` to indicate that the weak comparison algorithm should be used.

:arrow_down: 

An ASCII header value to parse ETags from. ETag values within may be prefixed by ``W/`` to indicate that the weak comparison function should be used.